src/pages/Layout/SideNav.tsx

```diff
@@ -0,0 +1,42 @@
+import Avatar from '@/components/common/Avatar/Avatar';
+
+import { FaBell, FaHome, FaPlusCircle } from 'react-icons/fa';
+import { useNavigate } from 'react-router-dom';
+
+import {
+  AvatarGroup,
+  Container,
+  ProfileAvatar,
+  ProfileName,
+  ProfileWrapper,
+  iconStyle,
+} from './SideNav.style';
+
+interface SideNav {
+  name?: string;
+}
+
+const SideNav = ({ name = '무명' }: SideNav) => {
+  const navigate = useNavigate();
+
+  return (
+    <Container>
+      <AvatarGroup>
+        {Array(8)
+          .fill(null)
+          .map((_, index) => (
+            <Avatar key={index} avatarShape="normal" onClick={() => navigate('/club/home')} />
+          ))}
+      </AvatarGroup>
+      <FaPlusCircle className={iconStyle} onClick={() => navigate('/club/create')} />
+      <FaHome className={iconStyle} onClick={() => navigate('/')} />
+      <FaBell className={iconStyle} onClick={() => alert('알림페이지 준비 중')} />
+      <ProfileWrapper onClick={() => navigate('/profile')}>
+        <ProfileAvatar avatarShape="rectangle" />
```

Comment:
p2;
스타일을 상속하기 보다는 컴포넌트를 바로 import해서 가져오면 될것 같아요!
```suggestion
        <Avatar avatarShape="rectangle" />
```

Reply:
넵 적용했습니다!